test/fail_compilation/cppmangle.d

```diff
@@ -15,9 +16,10 @@ extern(C++, "0num")
 {
 }
 
-extern(C++, "std", )
```

Comment:
Is this no longer an error or can the test be kept?

Reply:
Ok, but it might be interesting to verify whether the tuple is allowed to have a trailing comma or not.

Reply:
Agreed with Rainers here, can you add a trailing comma somewhere in your `cppmangle` tests ?

Reply:
Are you sure? The fail is in the parser, not semantic... so it can't be in this file with the rest of these semantic errors. I need to add a whole new file to the tester just to test that one thing, which should be tested by whatever already tests `parseCondExp()`...
I can do it, I just don't know why, and I have to create a whole new file just for that one test.

Reply:
Hum, right, because you don't actually pass a tuple, so the point in moot. I guess when we fix this code (I see this state as temporary), we'll fix it. At the moment basically any usage of the feature needs to be surrounded by parenthesis.

Reply:
There are expressions that aren't ambiguous with an identifier.
`extern(C++, "hello" ~ "world")` doesn't need parens for instance.